configs/serviceops/types.yml

```diff
@@ -0,0 +1,4 @@
+types:
+- name: global/types/GenericListCheckerType
```

Comment:
For now I have removed the namespace completely. scoping is a different topic that needs to be discussed in a general context for all types, templates. How about we can go through it when we are reviewing all the samples together as a group.